MAJ Frictionless et ajout template PR

Miryad3108 opened the pull request:
MAJ Frictionless vers 4.26.0 pour vérifier la cohérence des exemples du schéma
Ajout template Pull Request

Miryad3108 committed:
MAJ Frictionless et ajout template PR

MAJ Frictionless vers 4.26.0 pour vérifier la cohérence des exemples du schéma
Ajout template Pull Request

Miryad3108 committed:
Update requirements.txt

Miryad3108 committed:
Correction example code_com

Correction de l'exemple fourni pour le code commune qui n'était pas correct

Miryad3108 committed:
Nouvelle version schéma

Nouvelle version schéma suite aux modifications de l'exemple du champ code_com dans schema.json

AntoineAugusti commented on schema.json:
```diff
@@ -238,7 +238,7 @@
          "name": "code_com",
          "description": "Code INSEE de la commune où se trouve le stationnement vélo",
-         "example": "n71729735",
+         "example": "67023",
```
Tu devrais pouvoir ajouter un custom check comme https://github.com/etalab/lieux-covoiturage/blob/416a72214916f15b0f6c0f8e2ffff24015f6bf1f/schema.json#L275-L282

Miryad3108 replied:
Il y a besoin d'ajouter un custom check si il y a un pattern dans la description du champ ?

AntoineAugusti replied:
Oui c'est préférable. La regex ne garantit pas que le code INSEE existe

Miryad3108 replied:
C'est bon ce que je viens d'ajouter ?

Miryad3108 replied:
C'est supprimé 
Tu peux vérifier et valider la PR ?

AntoineAugusti commented on schema.json:
```diff
@@ -32,8 +32,8 @@
    "created": "2021-03-10",
-   "lastModified": "2021-06-30",
-   "version": "1.0.0",
+   "lastModified": "2022-03-01",
```
J'ai vu qu'il y avait plusieurs fichiers avec des liens vers `master`, ptet modifier ça pour inclure la version

Miryad3108 replied:
C'est fait

Miryad3108 committed:
Correction des liens

Correction des liens vers les exemples valides, invalides dans schema.json
suppression des liens qui menaient vers une page 404 dans le README

Miryad3108 committed:
Modification version du schéma

Modification version du schéma suite à une erreur de versionnage

Miryad3108 committed:
Ajout vérification des codes INSEE

Miryad3108 commented on schema.json:
```diff
@@ -510,5 +510,13 @@
          }
       }
    ],
+   "custom_checks":[
```
@AntoineAugusti c'est bon comme ça ?

AntoineAugusti replied:
Je me suis trompé, il n'y a pas de [custom check](https://git.opendatafrance.net/validata/validata-core/-/tree/master/validata_core/custom_checks) pour INSEE, seulement pour SIRET et SIREN. Tu peux supprimer ceci. Navré.

Miryad3108 replied:
C'est supprimé 
Tu peux vérifier et valider ?

Miryad3108 committed:
Update schema.json

AntoineAugusti reviewed (approved):
Bon boulot ! 👏
